content/en/docs/source.md

```diff
@@ -191,6 +191,48 @@ cmake -DUSE_BUNDLED_DEPS=On -DMUSL_OPTIMIZED_BUILD=On ..
 make falco
 ```
 
+## openSUSE
+
+### Dependencies
+
+```bash
+zypper -n install gcc gcc-c++ git-core cmake libjq-devel ncurses-devel yaml-cpp-devel libopenssl-devel libcurl-devel c-ares-devel protobuf-devel grpc-devel patch which automake autoconf libtool libelf-devel
+```
+
+### Build Falco
+
+```bash
+git clone https://github.com/falcosecurity/falco.git
+cd falco
+mkdir -p build
+cd build
+cmake -DUSE_BUNDLED_DEPS=ON ..
```

Comment:
```suggestion
cmake ..
```

No need to use bundled deps since you're installing them into your system with the `zypper` command above.

Quickly tried it out and it works.